src/librustc_data_structures/sync.rs

```diff
@@ -181,46 +199,40 @@ cfg_if! {
             ($($blocks:tt),*) => {
                 // We catch panics here ensuring that all the blocks execute.
                 // This makes behavior consistent with the parallel compiler.
-                let mut panic = None;
+                let panic = ::rustc_data_structures::sync::Lock::new(None);
```

Comment:
Can we use `$crate` here? That should make it work anywhere.

Reply:
Sure.